src/openforms/forms/tests/test_api_forms.py

```diff
@@ -1085,8 +1085,10 @@ def setUpTestData(cls):
         )
 
     def test_detail_shows_translated_values_based_on_request_header(self):
-
+        self.en_form.translation_enabled = True
+        self.en_form.save()
```

Comment:
Should this live in the `make_translated` decorator?

Reply:
hrm no, because `make_translated` also works on models/factories other than `Form` while this field is only scoped to the `Form` model.

I'll break out the data set up into this test method rather than rely on `setUpTestData` though, because this is not a good pattern.